{{cookiecutter.project_slug}}/clients/vue3/src/main.css

```diff
@@ -1,5 +1,5 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
```

Comment:
**Info:** This syntax change may not be truly required here. However, opted to keep it consistent with what the Tailwind v3 installation docs indicate:
https://tailwindcss.com/docs/using-with-preprocessors